src/veritysetup/veritysetup.c

```diff
@@ -116,6 +185,42 @@ static int parse_options(const char *options) {
                                 return log_error_errno(SYNTHETIC_ERRNO(EINVAL), "hash-offset= expects a 512-byte aligned value.");
 
                         arg_hash_offset = off;
+                } else if ((val = startswith(word, "salt="))) {
+
+                        if (!string_is_safe(val))
+                                return log_error_errno(SYNTHETIC_ERRNO(EINVAL), "salt= is not valid.");
+
+                        if (isempty(val)) {
+                                arg_salt = mfree(arg_salt);
+                                arg_salt_size = 32;
+                        } else if (streq(val, "-")) {
+                                arg_salt = mfree(arg_salt);
+                                arg_salt_size = 0;
+                        } else {
```

Comment:
"-" syntax needs man page update

Reply:
done

I mentioned this: "<literal>-</literal>is the special value for empty."

I hope it is clear enough.